management_interface/management_interface/utils.py

```diff
@@ -0,0 +1,7 @@
+from django.core.exceptions import ValidationError
+
+
+def validated_email_domain(email: str, domain: str, error_message: str):
```

Comment:
I don't think we need this at all; see the `SecureEmailValidator`

Reply:
Removed.